addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -245,6 +255,16 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         }
     }
 
+    private boolean hasReportUri(List<String> cspOptions) {
+        for (String csp : cspOptions) {
+            String lowerCsp = csp.toLowerCase(Locale.ROOT);
+            if (lowerCsp.contains("report-uri") || lowerCsp.contains("report-to")) {
```

Comment:
Should check just `report-uri`? The `report-to` does not seem to cause an exception(?).